### PR TITLE
FSET-719 set etray passmarks

### PR DESCRIPTION
**Pull request by benjstephenson**

### CODE REVIEW

File: app/controllers/Phase2PassMarkSettingsController.scala [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  val uuidFactory: UUIDFactory
+
+  def create = Action.async(parse.json) { implicit request =>
+    withJsonBody[Phase2PassMarkSettings] { passMarkSettings => {
+        val newVersionUUID = uuidFactory.generateUUID()
+        val newPassMarkSettings = passMarkSettings.copy(version = newVersionUUID, createDate = DateTime.now())
+        for {
+          createResult <- passMarkService.createPassMarkSettings(newPassMarkSettings)
+        } yield {
+          auditService.logEvent("PassMarkSettingsCreated", Map(
```
Comment (satishkamatam) [inline app/controllers/Phase2PassMarkSettingsController.scala]: Do we need to be specific in creating the event PassMarkSettingsCreated -> Phase1 or Phase2

File: app/model/exchange/passmarksettings/PassMarkTestTypes.scala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model.exchange.passmarksettings
+
+import play.api.libs.json.{ Format, JsString, JsSuccess, JsValue }
+import reactivemongo.bson.{ BSON, BSONHandler, BSONString }
+
+object PassMarkTestTypes extends Enumeration {
```
Comment (satishkamatam) [inline app/model/exchange/passmarksettings/PassMarkTestTypes.scala]: Shall we delete this if not used

File: app/controllers/Phase2PassMarkSettingsController.scala (hunk 1 above)
Comment (benjstephenson) [inline app/controllers/Phase2PassMarkSettingsController.scala]: We probably should - I'll update the phase 1 controller too.

File: app/model/exchange/passmarksettings/PassMarkTestTypes.scala (hunk 2 above)
Comment (benjstephenson) [inline app/model/exchange/passmarksettings/PassMarkTestTypes.scala]: Yes

**Pull request merged**